Cargo.toml

```diff
@@ -34,6 +35,8 @@ mimalloc = { version = "0.1.48", default-features = false, features = [
   "secure",
 ] }
 pfctl = "0.7.0"
+multiaddr = "0.18.2"
+notify = { version = "8.2.0", features = ["crossbeam-channel"] }
```

Comment:
Please, run `cargo shear --fix` in the environment at the end of the PR preparation cycle, to remove unneeded deps.